decidim-elections/lib/decidim/elections/admin_engine.rb

```diff
@@ -58,11 +58,14 @@ def self.participatory_space_endpoints
             next unless component
 
             link = Decidim::EngineRouter.admin_proxy(component).trustees_path(locale: I18n.locale)
+
+            has_election_components = current_participatory_space.components.select { |c| c.manifest_name == "elections" }.any?
+
             menu.add_item :trustees,
                           I18n.t("trustees", scope: "decidim.elections.admin.menu"),
                           link,
+                          if: allowed_to?(:manage, :trustees) || has_election_components && current_user.admin?,
```

Comment:
should not be this check something like : 

```
has_election_components && ( allowed_to?(:manage, :trustees) || current_user.admin?),
```

I am thinking that if an user is allowed to manage trustees, then only the first if condition will be applied, and the component check may not be added to question.

Reply:
I think it would be clearer as @alecslupu mentions above.

Also, I would put the `current_user.admin?` rule to the permissions instead, it seems redundant here. Or is there a particular reason why you added the admin check here @josepjaume ?

Reply:
Hi! Thanks for your comments! My first instinct was to fix the permissions logic but what I found is that we're in a special case, as this specific permission is defined on the `elections` component (but not on the process). That was the source of the issue - the item was only visible on the "elections" component.

Furthermore, we want to show that element in any participatory process if it has an elections component.

Can you think of a clean way to put that in terms of permissions? I couldn't find it 🤔 . In the meanwhile, I'll do that @alecslupu suggested 👍

Reply:
Got it @josepjaume, thanks for the explanation. I think then your approach is fine.

What about @alecslupu suggestion above? I think it would be slightly easier to read (although it won't change any logic).

Reply:
Oh, of course! Just changed it :)